Misc/NEWS.d/next/Library/2025-06-25-14-13-39.gh-issue-135661.idjQ0B.rst

```diff
@@ -0,0 +1,23 @@
+Fix parsing start and end tags in :class:`html.parser.HTMLParser`.
+
+* Whitespaces no longer accepted between ``</`` and the tag name.
+  E.g. ``</ script>`` does not end the script section.
+
+* Vertical tabulation (``\v``) and non-ASCII whitespaces no longer recognized
+  as whitespaces. The only whitespaces are ``\t\n\r\f`` and space.
+
+* Null character (U+0000) no longer ends the tag name.
+
+* End tag can have attributes and slashes after tag name. It no longer ends
+  after the first ``>`` in quoted attribute value. E.g. ``</script/foo=">"/>``.
+
+* Multiple slashes and whitespaces between the last attribute and closing ``>``
+  are now accepted in both start and end tags. E.g. ``<a foo=bar/ //>``.
```

Comment:
```suggestion
* Multiple slashes and whitespaces between the last attribute and closing ``>``
  are now parsed as comments in both start and end tags. E.g. ``<a foo=bar/ //>``.
```
Are these parsed as comment too or ignored?  I would change the phrasing as suggested, and possibly adding more tests for this case (I don't think I saw them).

Reply:
They are ignored.

They are tested in test_slashes_in_endtag.